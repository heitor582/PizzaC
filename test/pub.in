G
37
#
1
Calabresa
35.50
G
s
2
3
Calabresa
2
40.00
0
5
Queijo
2.50
6
7
Queijo
2
3.00
0
3
Calabresa
3
1
4
1
0
8
Queijo
2
9
1
n
4
Calabresa
0