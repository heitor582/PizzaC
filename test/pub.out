[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2JMenu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Op√ß√£o inv√°lida. Tente novamente.

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Op√ß√£o inv√°lida. Tente novamente.

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Op√ß√£o inv√°lida. Tente novamente.

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Cadastrar uma pizza selecionado.
Digite o sabor da pizza: Digite o pre√ßo da pizza: Digite o tamanho da pizza: Voce nao tem ingredientes cadastrados deseja salvar mesmo assim? (s/n): 
Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Ver todas as pizzas selecionado.
Pizza ID: 1
Sabor: Calabresa
Sabor: G
Pre√ßo: R$35.50



Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Atualizar uma pizza selecionado.
Digite o sabor da pizza que quer alterar: Menu de Op√ß√µes Atualiza√ß√£o da pizza:
0 - Sair
1 - Atualizar sabor da pizza
2 - Atualizar valor da pizza
3 - Adicionar ingredientes √† pizza
4 - Remover ingredientes da pizza
Escolha uma op√ß√£o: Op√ß√£o selecionada: Atualizar valor da pizza.
Digite o pre√ßo da pizza: 
Menu de Op√ß√µes Atualiza√ß√£o da pizza:
0 - Sair
1 - Atualizar sabor da pizza
2 - Atualizar valor da pizza
3 - Adicionar ingredientes √† pizza
4 - Remover ingredientes da pizza
Escolha uma op√ß√£o: Saindo da atualiza√ß√£o da pizza...

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Cadastrar um ingrediente selecionado.
Digite o nome do ingredient: Digite o pre√ßo extra do ingredient: 
Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Ver todos os ingredientes selecionado.
Ingrediente ID: 1
Nome: Queijo
Pre√ßo: R$2.50

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Atualizar um ingrediente selecionado.
Digite o ingrediente que quer alterar: Menu de Op√ß√µes Atualiza√ß√£o do ingrediente:
0 - Sair
1 - Atualizar nome do ingrediente
2 - Atualizar valor do ingrediente
Escolha uma op√ß√£o: Op√ß√£o selecionada: Atualizar valor do ingrediente.
Digite o novo pre√ßo do ingrediente: 
Menu de Op√ß√µes Atualiza√ß√£o do ingrediente:
0 - Sair
1 - Atualizar nome do ingrediente
2 - Atualizar valor do ingrediente
Escolha uma op√ß√£o: Saindo da [3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2J[3J[H[2Jatualiza√ß√£o do ingrediente...

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Atualizar uma pizza selecionado.
Digite o sabor da pizza que quer alterar: Menu de Op√ß√µes Atualiza√ß√£o da pizza:
0 - Sair
1 - Atualizar sabor da pizza
2 - Atualizar valor da pizza
3 - Adicionar ingredientes √† pizza
4 - Remover ingredientes da pizza
Escolha uma op√ß√£o: Op√ß√£o selecionada: Adicionar ingredientes √† pizza.
Ingrediente ID: 1
Nome: Queijo
Pre√ßo: R$3.00
Digite o id do ingrediente desejado: 
Menu de Op√ß√µes Atualiza√ß√£o da pizza:
0 - Sair
1 - Atualizar sabor da pizza
2 - Atualizar valor da pizza
3 - Adicionar ingredientes √† pizza
4 - Remover ingredientes da pizza
Escolha uma op√ß√£o: Op√ß√£o selecionada: Remover ingredientes da pizza.
Digite o id do ingrediente desejado: 
Menu de Op√ß√µes Atualiza√ß√£o da pizza:
0 - Sair
1 - Atualizar sabor da pizza
2 - Atualizar valor da pizza
3 - Adicionar ingredientes √† pizza
4 - Remover ingredientes da pizza
Escolha uma op√ß√£o: Saindo da atualiza√ß√£o da pizza...

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Deletar um ingrediente selecionado.
Digite o nome do ingrediente para deletar: Ingrediente 'Queijo' exclu√≠do com sucesso!

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Ver todas as pizzas selecionado.
Pizza ID: 1
Sabor: Calabresa
Sabor: G
Pre√ßo: R$40.00



Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Vender pizza selecionado.
Pizza ID: 1
Sabor: Calabresa
Sabor: G
Pre√ßo: R$40.00


Selecione o ID da pizza que quer vender: Deseja adicionar ingredientes extras? (s/n): 
Resumo do pedido:
Pizza: Calabresa (R$ 40.00)
Adicionais: R$ 0.00
Total a pagar: R$ 40.00

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Deletar uma pizza selecionado.
Digite o sabor da pizza que quer deletar: Pizza 'Calabresa' exclu√≠do com sucesso!

Menu de Op√ß√µes:
0 - Sair
1 - Cadastrar uma pizza
2 - Ver todas as pizzas
3 - Atualizar uma pizza
4 - Deletar uma pizza
5 - Cadastrar um ingrediente
6 - Ver todos os ingredientes
7 - Atualizar um ingrediente
8 - Deletar um ingrediente
9 - Vender pizza
Escolha uma op√ß√£o: Saindo do programa...
